projects/1.18/assets/stoneworks/stoneworks/lang/zh_cn.json

```
@@ -0,0 +1,584 @@
+  "block.stoneworks.polished_tuff": "磨制凝灰岩",
+  "block.stoneworks.polished_tuff_slab": "磨制凝灰岩台阶",
+  "block.stoneworks.polished_tuff_stairs": "磨制凝灰岩墙",
+  "block.stoneworks.polished_tuff_wall": "磨制凝灰岩墙",
+  "block.stoneworks.prismarine_brick_slab": "海晶石砖台阶",
+  "block.stoneworks.prismarine_brick_stairs": "海晶石砖楼梯",
+  "block.stoneworks.prismarine_brick_wall": "海晶石砖墙",
+  "block.stoneworks.prismarine_bricks": "海晶石砖",
+  "block.stoneworks.prismarine_paver_slab": "海晶石地砖台阶",
+  "block.stoneworks.prismarine_paver_stairs": "海晶石地砖楼梯",
+  "block.stoneworks.prismarine_paver_wall": "海晶石地砖墙",
+  "block.stoneworks.prismarine_pavers": "海晶石地砖",
+  "block.stoneworks.prismarine_pillar": "海晶石柱",
+  "block.stoneworks.prismarine_plate_slab": "切制海晶石台阶",
+  "block.stoneworks.prismarine_plate_stairs": "切制海晶石楼梯",
+  "block.stoneworks.prismarine_plate_wall": "切制海晶石墙",
+  "block.stoneworks.prismarine_plates": "切制海晶石",
+  "block.stoneworks.prismarine_tile_slab": "台阶竹制海晶石台阶",
+  "block.stoneworks.prismarine_tile_stairs": "竹制海晶石楼梯",
+  "block.stoneworks.prismarine_tile_wall": "竹制海晶石墙",
+  "block.stoneworks.prismarine_tiles": "竹制海晶石",
+  "block.stoneworks.purpur_brick_slab": "紫珀块砖台阶",
+  "block.stoneworks.purpur_brick_stairs": "紫珀块砖楼梯",
+  "block.stoneworks.purpur_brick_wall": "紫珀块砖墙",
+  "block.stoneworks.purpur_bricks": "紫珀块砖",
+  "block.stoneworks.purpur_paver_slab": "紫珀块地砖台阶",
+  "block.stoneworks.purpur_paver_stairs": "紫珀块地砖楼梯",
+  "block.stoneworks.purpur_paver_wall": "紫珀块地砖墙",
+  "block.stoneworks.purpur_pavers": "紫珀块地砖",
+  "block.stoneworks.purpur_shingle_slab": "瓦制紫珀块台阶",
+  "block.stoneworks.purpur_shingle_stairs": "瓦制紫珀块楼梯",
+  "block.stoneworks.purpur_shingle_wall": "瓦制紫珀块墙",
+  "block.stoneworks.purpur_shingles": "瓦制紫珀块",
+  "block.stoneworks.purpur_tile_slab": "竹制紫珀块台阶",
+  "block.stoneworks.purpur_tile_stairs": "竹制紫珀块楼梯",
+  "block.stoneworks.purpur_tile_wall": "竹制紫珀块墙",
+  "block.stoneworks.purpur_tiles": "竹制紫珀块",
+  "block.stoneworks.quartz_paver_slab": "切制石英块台阶",
+  "block.stoneworks.quartz_paver_stairs": "切制石英块楼梯",
+  "block.stoneworks.quartz_paver_wall": "切制石英块墙",
+  "block.stoneworks.quartz_pavers": "切制石英块",
+  "block.stoneworks.quartz_plate_slab": "切制石英块台阶",
+  "block.stoneworks.quartz_plate_stairs": "切制石英块楼梯",
+  "block.stoneworks.quartz_plate_wall": "切制石英块墙",
+  "block.stoneworks.quartz_plates": "切制石英块",
+  "block.stoneworks.quartz_shingle_slab": "瓦制石英块台阶",
+  "block.stoneworks.quartz_shingle_stairs": "瓦制石英块楼梯",
+  "block.stoneworks.quartz_shingle_wall": "瓦制石英块墙",
+  "block.stoneworks.quartz_shingles": "瓦制石英块",
+  "block.stoneworks.quartz_tile_slab": "竹制石英块台阶",
+  "block.stoneworks.quartz_tile_stairs": "竹制石英块楼梯",
+  "block.stoneworks.quartz_tile_wall": "竹制石英块墙",
+  "block.stoneworks.quartz_tiles": "竹制石英块",
+  "block.stoneworks.raw_dark_prismarine": "原始暗海晶石",
+  "block.stoneworks.raw_dark_prismarine_slab": "原始暗海晶石台阶",
+  "block.stoneworks.raw_dark_prismarine_stairs": "原始暗海晶石楼梯",
+  "block.stoneworks.raw_dark_prismarine_wall": "原始暗海晶石墙",
```

```suggestion
  "block.stoneworks.raw_dark_prismarine": "粗糙暗海晶石",
  "block.stoneworks.raw_dark_prismarine_slab": "粗糙暗海晶石台阶",
  "block.stoneworks.raw_dark_prismarine_stairs": "粗糙暗海晶石楼梯",
  "block.stoneworks.raw_dark_prismarine_wall": "粗糙暗海晶石墙",
```
参照原版`粗铁块`/`Block of Raw Iron`修改，仅建议
如修改则请全文修改